setup1:
  ip_robot: "192.168.131.200"
  # port_robot: 80
  port_robot: 443
  name_robot: "ROB_1"
  task_robot: "T_ROB1"
  pos_corr_gain: 1
  max_speed_deviation: 300
  robot_port_number: 6511
  root_name: "single_yumi_base_link"
  tip_name: "gripper_base"
